### PR TITLE
Content/guidance key - Heading

### PR DESCRIPTION
**Pull request by andij**

### CODE REVIEW

File: src/content/docs/components/ns-selector.mdx [hunk 1]
```diff
@@ -81,7 +81,7 @@ Showcase a product, service or option which users can select. The `ns-selector`
 | Key | Field type | Guidelines |
 | :--- | :--- | :--- |
 | <span class="marker">A</span> | Label | The label should describe the selection being made using the most concise content possible. The recommended length is 21 characters maximum. |
-| <span class="marker">B</span> | Heading | The heading can provide an expanded description of the item being selected. The recommended length is between 3 and 21 characters. This heading should use a light font weight to allow the emphasis on the label. |
+| <span class="marker">B</span> | Heading | The heading can provide further description of the selected item. The recommended length is between 4 and 12 words, not exceeding 50 characters in total. |
```
Comment (MekalaNagarajan-Centrica) [inline src/content/docs/components/ns-selector.mdx:84]: `selected item` is not correct. `item to be selected`.

Comment (andij) [reply src/content/docs/components/ns-selector.mdx:84]: I'd like to suggest a rewrite of the first sentence:

From:
`The heading can provide an expanded description of the item being selected.`

To:
`A heading offers additional clarification to the label.`

Or:
`A heading provides further description to the label.`

Comment (RobTobias123) [reply src/content/docs/components/ns-selector.mdx:84]: The latter is better I think.

**Review by RobTobias123 — APPROVED** [review]
Reads well.

File: src/content/docs/components/ns-selector.mdx (hunk 1 above)
Comment (andij) [reply src/content/docs/components/ns-selector.mdx:84]: I've updated this sentence to the latter.

**Pull request merged**